conda-requirements-esp-tools-linux-64.conda-lock.yml

```diff
@@ -0,0 +1,4617 @@
+# This lock file was generated by conda-lock (https://github.com/conda-incubator/conda-lock). DO NOT EDIT!
```

Comment:
You might consider marking the checked-in conda-lock.yml files as generated as described in https://docs.github.com/en/repositories/working-with-files/managing-files/customizing-how-changed-files-appear-on-github so that they are by default not shown in PR diffs. You can still click a link to expand the diff but it explicitly says that the file is generated and you're probably not interested in it.

Reply:
Oh interesting. I'll look into this.